Source/JavaScriptCore/b3/air/AirCustom.h

```diff
@@ -131,26 +132,38 @@ struct CCallCustom : public CommonCustomBase<CCallCustom> {
     template<typename Functor>
     static void forEachArg(Inst& inst, const Functor& functor)
     {
-        Value* value = inst.origin;
+        CCallValue* value = inst.origin->as<CCallValue>();
 
         unsigned index = 0;
 
-        functor(inst.args[index++], Arg::Use, GP, pointerWidth()); // callee
-        
-        if (value->type() != Void) {
-            functor(
-                inst.args[index++], Arg::Def,
+        auto next = [&](Arg::Role role, Bank bank, Width width) {
+            functor(inst.args[index++], role, bank, width);
+        };
+
+        next(Arg::Use, GP, pointerWidth()); // callee
+
+        for (size_t n = cCallResultCount(value); n; --n) {
+            next(
+                Arg::Def,
                 bankForType(value->type()),
-                widthForType(value->type()));
+                is32Bit() && value->type() == Int64 ? Width32 : widthForType(value->type()));
```

Comment:
This feels wrong to even emit code like this. I feel like we should be emitting code in Air that doesn't deal with 64-bit ints at all. So instead, it should just be a tuple of int32 or something like that